server/statistics/hot_peer_cache.go

```diff
@@ -209,41 +209,50 @@ func (f *hotPeerCache) CheckRegionFlow(region *core.RegionInfo) (ret []*HotPeerS
 
 func (f *hotPeerCache) IsRegionHot(region *core.RegionInfo, hotDegree int) bool {
 	switch f.kind {
-	case WriteFlow:
+	case PeerCache:
 		return f.isRegionHotWithAnyPeers(region, hotDegree)
-	case ReadFlow:
+	case LeaderCache:
 		return f.isRegionHotWithPeer(region, region.GetLeader(), hotDegree)
 	}
 	return false
 }
 
-func (f *hotPeerCache) CollectMetrics(typ string) {
+func (f *hotPeerCache) CollectMetrics() {
 	for storeID, peers := range f.peersOfStore {
 		store := storeTag(storeID)
 		thresholds := f.calcHotThresholds(storeID)
-		hotCacheStatusGauge.WithLabelValues("total_length", store, typ).Set(float64(peers.Len()))
-		hotCacheStatusGauge.WithLabelValues("byte-rate-threshold", store, typ).Set(thresholds[byteDim])
-		hotCacheStatusGauge.WithLabelValues("key-rate-threshold", store, typ).Set(thresholds[keyDim])
-		// for compatibility
-		hotCacheStatusGauge.WithLabelValues("hotThreshold", store, typ).Set(thresholds[byteDim])
+		hotCacheStatusGauge.WithLabelValues("total_length", store, f.kind.String()).Set(float64(peers.Len()))
```

Comment:
I find the formats of labels are not consistent

Reply:
em, do you mean it should be `total-length`?

Reply:
Yes. We have `byte-rate-threshold`, `hotThreshold`, and `total_length` three different kinds of format.